packages/flutter_tools/test/general.shard/android/gradle_errors_test.dart

```diff
@@ -913,10 +913,10 @@ A problem occurred evaluating project ':app'.
           '│ To fix this issue, replace the following content:                                │\n'
           '│ /android/build.gradle:                                                           │\n'
           "│     - classpath 'com.android.tools.build:gradle:<current-version>'               │\n"
```

Comment:
These 2 lines have double quote instead of single quote? can you fix them while you are here?

Reply:
From what I can tell this was actually done intentionally, because the line itself contains single quotes:
```
classpath 'com.android.tools.build:gradle:<current-version>'
```

Reply:
ah that makes sense I didnt catch that.